v2/stack_analyses.yaml

```diff
@@ -0,0 +1,469 @@
+          properties: {}
+        pull_requests:
+          type: object
+          properties: {}
+        dependent_repos:
+          type: integer
+        open_issues_count:
+          type: integer
+        latest_release_duration:
+          type: string
+        forks_count:
+          type: integer
+        contributors:
+          type: integer
+        size:
+          type: string
+        stargazers_count:
+          type: integer
+        used_by:
+          type: array
+          items:
+            type: string
+        dependent_projects:
+          type: integer
+      description: Github details
+    PackageDetailsBase:
+      title: PackageDetailsBase
+      allOf:
+        - $ref: '#/components/schemas/Package'
+        - type: object
+          properties:
+            latest_version:
+              type: string
+            github:
+              $ref: '#/components/schemas/GitHubDetails'
+            licenses:
+              type: array
+              items:
+                type: string
+            topic_list:
+              type: array
+              items:
+                type: string
+            ecosystem:
+              type: string
+            public_vulnerabilities_count:
+              type: integer
+            private_vulnerabilities_count:
+              type: integer
+            affected_direct_deps:
+              type: array
+              description: Lists all direct dependencies of this package
+              items:
+                type: object
+                properties:
+                  package:
+                    type: string
+                  version:
+                    type: string
+            osio_user_count:
+              type: integer
+          description: Details of the package
+    PackageDetailsForRegisteredUser:
+      title: PackageDetailsForRegisteredUser
+      allOf:
+        - $ref: '#/components/schemas/PackageDetailsBase'
+        - type: object
+          properties:
+            public_vulnerabilities:
+              type: array
+              description: Publicly known vulnerability details
+              items:
+                $ref: '#/components/schemas/VulnerabilityForRegisteredUser'
+            private_vulnerabilities:
+              type: array
+              description: Private vulnerability details, available only to registered
+                users
+              items:
+                oneOf:
+                  - $ref: '#/components/schemas/VulnerabilityForRegisteredUser'
+          description: Details of the package for registered users
+    PackageDetailsForFreeTier:
+      title: PackageDetailsForFreeTier
+      allOf:
+        - $ref: '#/components/schemas/PackageDetailsBase'
+        - type: object
+          properties:
+            public_vulnerabilities:
+              type: array
+              description: Publicly known vulnerability details
+              items:
+                $ref: '#/components/schemas/VulnerabilityForFreeTier'
```

Comment:
Why is this field under public_vulnerabilities ??  Vulnabilities for free tier include complete list & details of public vulnerabilities + count and limited fields for private vulnerabilities.